db/db.go

```diff
@@ -18,7 +19,8 @@ var (
 )
 
 func GetImageFilePath(imageName string) string {
```

Comment:
if the issue is in the call for `VBoxManage.exe`, then I think srounding the output of this function with the double quote or quote char might be suitable only there, see https://github.com/mhewedy/vermin/blob/2ee8d2b7e8fb87d7fafa07ccda52dd44594892b4/hypervisor/virtualbox/virtualbox.go#L41

check here for places where this function is being called 
https://github.com/search?q=repo%3Amhewedy%2Fvermin%20GetImageFilePath&type=code
I am not sure if changing this function will have no effect on all these places.

Reply:
updated to just vboxmanage

tested using dlv